### PR TITLE
Jd pull requests - Added remaining sections of the page

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
Updated index.html / mobile_first.css
Completed the sections that were missing from the main template. Fixing the styles for those sections and added one last img for the last section background.

Please, let me know if I missed to add something or if I did great. It's been plenty of hours of coding these days and I learned a lot but didn't sleep enough jeje 😅.

I'll patiently wait for a review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
                                 <img src="img/social media icons.png" alt="github link">
-                            </a></li>
+                            </a>
+                        </li>
                     </ul>
                 </div>
                </footer>
             </section>
     </div>
+    <div id="my-recent-works">
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] For this section, kindly make the container of the section a `section` tag. This is in line of having a semantically well coded project and helps keep your project professional

File: css/mobile_first.css [hunk 2]
```diff
@@ -116,22 +135,392 @@ body {
...
+  line-height: 25px;
 }
 
-footer h3 {
-  font-style: normal;
+#my-recent-works {
+  margin-top: -28px;
+  text-align: center;
+  align-content: center;
+}
```
Comment (Shedrack-Sunday) [inline css/mobile_first.css]: - [ ] The main container of the projects should have a `display:grid` property. This was stated in the project requirements, and made so to help you understand how to use the `grid` property to display and structure content 👍 

## Instruction:
image

File: index.html [hunk 3]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+        <h1 class="recent-works-tittle">My Recent <br> Works</h1>
```
Comment (Shedrack-Sunday) [inline index.html]: - [  ] Only one `h1` tag should be used per project. This is in line with using the right semantics, which directly affects how the web crawls your website for example. So kindly use a `hx` element like `h2` `h3` for this section 👍

File: css/mobile_first.css [hunk 4]
```diff
@@ -116,22 +135,392 @@ body {
...
+  border: 0.5px solid #d0d9d4;
+  text-align: center;
+}
+
+.project-2 {
+  margin: 20px 24px 0 24px;
+  border-radius: 8px;
+  border: 0.5px solid #d0d9d4;
+  text-align: center;
+}
```
Comment (Shedrack-Sunday) [inline css/mobile_first.css]: - [ ] Kindly make sure the projects listed are aligned with the flexbox. The use of the flex display was introduced to help you understand how to effectively use such which is an industry best practice

File: index.html [hunk 5]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+              <h3>
+                Hello-Microverse-Project <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
+                </div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] For this section, the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") is a list `ul` and each of them a `li`

## Instructions:
image

File: index.html [hunk 6]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Please refer to line `63` on the use of proper semantics for this section 👍

File: index.html [hunk 7]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+                    <img src="img/Icon Langguage.png" alt="first project image" class="icon_language">
+                    
+              <h3 id="h3-lang">
+                Languages
+              </h3>
+                <div class="am-languages-grid">                    
+                        <div class="am-lang-list am1"><strong>Html5</strong></div>
+                        <div class="am-lang-list am2"><strong>Css3</strong></div>
+                        <div class="am-lang-list am3"><strong>Git</strong></div>
+                        <div class="am-lang-list am4"><strong>Github</strong></div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Please refer to line `63` on the use of proper semantics for this section 👍

File: index.html [hunk 8]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+                        
+                  <h3 id="h3-lang">
+                    Frameworks
+                  </h3>
+                    <div class="am-frameworks-grid">                    
+                            <div class="am-frame-list f1"><strong>Angular</strong></div>
+                            <div class="am-frame-list f2"><strong>React</strong></div>
+                            <div class="am-frame-list f3"><strong>Spring</strong></div>
+                            <div class="am-frame-list f4"><strong>ASP.NET</strong></div>
+                            <div class="am-frame-list f5"><strong>NodeJS</strong></div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Please refer to line `63` on the use of proper semantics for this section 👍

File: index.html [hunk 9]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+                        
+                  <h3 id="h3-lang">
+                    Skills
+                  </h3>
+                    <div class="am-skills-grid">
+                            <div class="am-frame-list sk1"><strong>Database Management</strong></div>                    
+                            <div class="am-frame-list sk2"><strong>Version Control</strong></div>
+                            <div class="am-frame-list sk3"><strong>Web Development</strong></div>
+                            <div class="am-frame-list sk4"><strong>UX/UI</strong></div>
+                            <div class="am-frame-list sk5"><strong>SQL</strong></div>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Please refer to line `63` on the use of proper semantics for this section 👍

File: index.html [hunk 10]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+                        <div class="lang-list l4"><strong>Github</strong></div>
+                </div>
+                <div class="external_link">
+                    <a href="#">See Project</a>
+                </div>
+            </div>
+        </div>       
+    </div>
+    <div id="about-me">
+        <div class="about-me-bg">
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] For this section, The container of the section should be a  `section` tag. This is semantically correct as a best practice, and helps the crawling of your webpage easier 👍

File: index.html [hunk 11]
```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
...
+        <h1 class="about-me-tittle">About me</h1>
```
Comment (Shedrack-Sunday) [inline index.html]: - [ ] Using only one `H1` creates a hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about. Kindly use a `hx` instead 👍

File: css/mobile_first.css [hunk 12]
```diff
@@ -116,22 +135,392 @@ body {
...
+  font-weight: bold;
+  font-size: 18px;
   color: var(--gris);
-  line-height: 44px;
+  line-height: 25px;
 }
 
-footer h3 {
-  font-style: normal;
+#my-recent-works {
```
Comment (Shedrack-Sunday) [inline css/mobile_first.css]: - [ ] Kindly adjust the margin at the top of the works section to look visually appealing and give a better user experience. This is also important in sticking to requirements 

## Your design:
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @JuanDiegoGuerra      👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 

  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented which majorly has to do with semantics and a bit of styling 


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by LionRouge1 — APPROVED** [review]
Hi @JuanDiegoGuerra, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] Please You must stick to the design as much as possible (e.g., font, colors, images, text, margins). Your website background color and font are different from the design color.

### Your website
Screenshot from 2023-03-25 08-14-28

### How it suppose to look like
227675422-96a6da0e-a729-4c56-baaa-f4bc659cc1d8

This is a requirement of the project but the first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change.

****You can now get some sleep, because you deserve it**  :sunglasses:**   🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by JuanDiegoGuerra** [comment]
@LionRouge1 Hello! Thank you for your feedback, I just woke up and I'm happy to read that I achieved the goal. I will also try to improve what you suggested me. I tried to make it as similar as possible and maybe in my perspective (screen sizes tested) looked good and I didn't notice the differences but I know I can do it better next time!

thank_you_kr

**Pull request merged**